crates/re_sdk/src/log_sink.rs

```diff
@@ -76,6 +76,50 @@ impl LogSink for BufferedSink {
     }
 }
 
+/// The storage used by [`MemorySink`]
+#[derive(Default, Clone)]
+pub struct MemorySinkStorage(std::sync::Arc<parking_lot::Mutex<Vec<LogMsg>>>);
+
+///
+impl MemorySinkStorage {
+    /// Lock the contained buffer
+    pub fn lock(&self) -> parking_lot::MutexGuard<'_, Vec<LogMsg>> {
```

Comment:
I don't see this being used outside this function - no need to leak internals imho
```suggestion
   fn lock(&self) -> parking_lot::MutexGuard<'_, Vec<LogMsg>> {
```

Reply:
Ahh good call -- went away once I moved `rrd_as_bytes` into this object.